soapui-testserver-api/src/main/java/com/smartbear/ready/recipe/TestStepNames.java

```diff
@@ -12,4 +14,20 @@
     public static final String DELAY_TYPE = "Delay";
     public static final String PROPERTIES_TYPE = "Properties";
     public static final String SOAP_MOCK_RESPONSE_TYPE = "SOAPMockResponse";
+
+    public static String createUniqueName(WsdlTestCase testCase, String testStepName) {
+        if (testCase.getTestStepByName(testStepName) == null) {
+            return testStepName;
+        }
+
+        String orgName = testStepName;
```

Comment:
Avoid abbreviations in variable names. Please use originalName and count rather than orgName and cnt.